views/components/Matrix.d3.js

```diff
@@ -311,7 +232,7 @@ export default class Matrix extends AbstractGraph {
       this.width,
       this.matrixWidth + this.maxYAxisWidth + margin.left
     )
-    this.zoom = d3
+    const zoom = d3
```

Comment:
curious about this becoming a constant

Reply:
it didn't need to be an attribute of the class since it wasn't used anywhere. Also after our sprint planning it seems like the zoom functionality is not desired functionality so maybe we can remove this altogther.